src/main/webapp/adminCertainOrder.jsp

```diff
@@ -122,30 +122,13 @@
             </h1>
             <p><fmt:message key="label.infoAboutOrder"/></p>
             <table class="table table-hover">
-                <thead>
+                <form action = "/admin/1/orders/1" method = "post">
                 <tr>
-                    <th><fmt:message key="label.name"/></th>
-                    <th><fmt:message key="label.phoneNumber"/></th>
-                    <th><fmt:message key="label.email"/></th>
-                    <th><fmt:message key="label.startDate"/></th>
-                    <th><fmt:message key="label.period"/></th>
-                    <th><fmt:message key="label.carModel"/></th>
-                    <th><fmt:message key="label.payment"/></th>
-                    <th><fmt:message key="label.status"/></th>
+                    <td><c:out value="${offer.status}"/></td>
+                    <td><input type="text" placeholder="new status" name="status"/></td>
```

Comment:
for status validation, probably here you can add type="number" required max=4

Reply:
It also should be done different way, but i'm not very good in frontend